app/scripts/account-import-strategies/index.js

```diff
@@ -3,7 +3,7 @@ import Wallet from 'ethereumjs-wallet'
 import importers from 'ethereumjs-wallet/thirdparty'
 import ethUtil from 'ethereumjs-util'
 
-const accountImporter = {
+export default {
```

Comment:
I think this is something we generally shouldn't do, as it makes importing specific elements awkward (e.g. this would prevent you from doing `import { importAccount } from './account-import-strategies'`

Easy to fix though:
```suggestion
export {
```